### PR TITLE
Show message for expired links

### PR DESCRIPTION
**Pull request by richtier**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] Change is accessible to the standards we subscribe to.
 - [x] Add a printscreen to the PR

image

### CODE REVIEW

File: enrolment/templates/enrolment/individual-collaborator-invite-expired.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+{% block content %}
+    {% with contact_url=services_urls.contact_us|add:'domestic/enquiries/' %}
+    <section class="container">
+        {% breadcrumbs 'Create a great.gov.uk account' %}
+            <a href="{{ services_urls.great_domestic }}">Home</a>
+        {% endbreadcrumbs %}
+        <div class="grid-row margin-top-60">
+            <div class="column-two-thirds">
+                <div id="success-message-container">
```
Comment (MichaelWalker) [inline enrolment/templates/enrolment/individual-collaborator-invite-expired.html]: but its an error message...

File: enrolment/templates/enrolment/individual-collaborator-invite-expired.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                    {% with description="This invitation has expired, please contact your business profile administrator to request a new invitation or <a href='"|add:contact_url|add:"'>contact us.</a>"|safe %}
```
Comment (MichaelWalker) [inline enrolment/templates/enrolment/individual-collaborator-invite-expired.html]: this seems like a slightly odd way to build a URL - do we not have a contant / route config that points at this?

File: enrolment/templates/enrolment/individual-collaborator-invite-expired.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            </div>
+        </div>
+    </section>
+    <section class="container">
+        <h2 class="heading-medium margin-0">Next steps</h2>
+        <p>You can now:</p>
+        <ul class="list list-bullet">
+            <li><a href="{{ contact_url }}">Contact us</a></li>
+        </ul>
+        <p>Return to the <a href="{% url 'index' %}">home page</p>
```
Comment (MichaelWalker) [inline enrolment/templates/enrolment/individual-collaborator-invite-expired.html]: Code is fine, but seems slightly odd to have a list of 1 thing, followed by another option that isn't in the list.

Is this deliberate?

File: enrolment/templates/enrolment/individual-collaborator-invite-expired.html (hunk 2 above)
Comment (richtier) [reply enrolment/templates/enrolment/individual-collaborator-invite-expired.html]: unfortunately, concatenation in the template is hard. I'll move it to the view

File: enrolment/templates/enrolment/individual-collaborator-invite-expired.html (hunk 3 above)
Comment (richtier) [reply enrolment/templates/enrolment/individual-collaborator-invite-expired.html]: it's reusing the existing `Next steps` "component" which we use in a few places.

while it's odd, it's portable for ease of copy and paste:)

**Pull request merged**